packages/shaders/src/shaders/waves.ts

```diff
@@ -56,8 +56,8 @@ void main() {
   float spacing = .02 + .98 * u_spacing;
   float shape = .5 + .5 * sin((shape_uv.y + offset) * PI / spacing);
 
-  float edge_width = .02 / (1. + abs(shape)) * (.001 + u_scale);
-  edge_width += .5 * max(0., u_softness);
```

Comment:
I think we had added this to avoid pixelation automatically

New version:

![Screenshot 2025-06-19 at 17 38 12](https://github.com/user-attachments/assets/9c3414de-40ff-46ab-882e-dd4b02768a36)

Current:

![Screenshot 2025-06-19 at 17 38 11](https://github.com/user-attachments/assets/b05cd735-f222-43f7-bfc9-fa706b1e1508)